main.go

```diff
@@ -107,7 +107,10 @@ func main() {
 
 	w := ui.NewWindow()
 
-	ui.SetDefaultRootFolder("ui/build")
+	if !ui.SetDefaultRootFolder("ui/build") {
+		println("The folder `./ui/build` does not exist. Please run `bun run --cwd ui build`")
+		return
+	}
```

Comment:
```suggestion
	if err := ui.SetDefaultRootFolder("ui/build"); err != nil {
		log.Fatalf("Ensure you have run `bun run --cwd ui build`. %v", err)
	}
```

I updated the go-webui dependecy to use the development version.
With the latest changes this wouldn't be a bool anymore.

Might be ready to merge after this